style.css

```diff
@@ -0,0 +1,94 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: 'Roboto', sans-serif;
+  font-size: 16px;
+  color: #333;
+  background-color: #fff;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  overflow-x: hidden;
+  overflow-y: auto;
+  width: 450px;
+  display: flex;
+  flex-direction: column;
+}
+
+nav {
+  background-color: #333;
+  padding: 20px;
+  color: #fff;
+  position: feixed;
+  width: 450px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.header .logo {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 4px;
+  min-width: 100px;
+  width: 100%;
+}
+
+i {
+  display: flex;
+  justify-items: flex-end;
+}
+
+.container {
+  background-color: #1c1a19;
+  display: flex;
+  justify-content: space-between;
+  align-self: center;
+  flex-wrap: wrap;
+  height: 580px;
+  width: 450px;
```

Comment:
- [ ] Going through your CSS, I can see you used mostly `pixels` for widths and heights. As stated in the project requirement `Use responsive values in your CSS rules, like percentages (i.e. width: 80%), instead of pixels`, I'll suggest you use responsive values as much as you can, this will help your website adapt and resize easily across multiple screen sizes. :wave: :+1: